tests/test_fxns.py

```diff
@@ -44,39 +43,39 @@ def fibronectin():
     return "fibronectin pdb"
 
 
-@pytest.fixture
-def get_registry():
-    return PathRegistry()
+# @pytest.fixture
```

Comment:
can you remove these, as they appear unnecessary here? (and everywhere else this is)

Reply:
sure